Client/Pages/Index.razor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Client.ViewModels;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+using Client.Services;
+using Client.Models;
+
+namespace Client.Pages
+{
+    public partial class Index
+    {
+        private bool isLoading = false;
+        public static IList<EncFile> fileList = new List<EncFile>();
+        private bool fileTooLarge = false;
```

Comment:
Good work. I like the idea of using FileUtilService combined with the codebehind page.